libs/ts/base-utils/src/evm/networks.ts

```diff
@@ -252,716 +252,1203 @@ export const networkMetadata = {
+    explorer: {
+      type: 'bitlayer',
+      webUrl: 'https://testnet-scan.bitlayer.org',
+      apiUrl: 'https://api-testnet.bitlayer.org/scan/api',
+    },
     currency: Currency.BTC,
   },
   'blast-mainnet': {
     chainId: 81457,
     isTestnet: false,
-    explorerUrl: 'https://blastscan.io',
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://explorer-testnet.unit0.dev',
+      apiUrl: 'https://explorer-testnet.unit0.dev/api',
+    },
     currency: Currency.ETH,
   },
   'blast-sepolia': {
     chainId: 168587773,
     isTestnet: true,
-    explorerUrl: 'https://sepolia.blastscan.io',
+    explorer: {
+      type: 'etherscan',
+      webUrl: 'https://sepolia.blastscan.io',
+      apiUrl: 'https://api-sepolia.blastscan.io/api',
+    },
     currency: Currency.ETH,
   },
   'boba-bnb-mainnet': {
     chainId: 56288,
     isTestnet: false,
-    explorerUrl: 'https://bnb.bobascan.com/',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://bnb.bobascan.com',
+      apiUrl: 'https://api.routescan.io/v2/network/mainnet/evm/56288/etherscan',
+    },
     currency: Currency.ETH,
   },
   'boba-bnb-testnet': {
     chainId: 9728,
     isTestnet: true,
-    explorerUrl: 'https://bnb.testnet.bobascan.com/',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://bnb.testnet.bobascan.com',
+      apiUrl: 'https://api.routescan.io/v2/network/testnet/evm/9728/etherscan',
+    },
     currency: Currency.ETH,
   },
   'boba-mainnet': {
     chainId: 288,
     isTestnet: false,
-    explorerUrl: 'https://bobascan.com',
     currency: Currency.ETH,
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://bobascan.com',
+      apiUrl: 'https://api.routescan.io/v2/network/mainnet/evm/288/etherscan',
+    },
   },
   'boba-sepolia': {
     chainId: 28882,
     isTestnet: true,
-    explorerUrl: 'https://sepolia.testnet.bobascan.com',
+    explorer: {
+      type: 'routescan',
+      webUrl: 'https://sepolia.testnet.bobascan.com',
+      apiUrl: 'https://api.routescan.io/v2/network/testnet/evm/28882/etherscan',
+    },
     currency: Currency.ETH,
   },
   'bsc-mainnet': {
     chainId: 56,
     isTestnet: false,
-    explorerUrl: 'https://bscscan.com',
+    explorer: {
+      type: 'etherscan',
+      webUrl: 'https://bscscan.com',
+      apiUrl: 'https://api.bscscan.com/api',
+    },
     currency: Currency.BNB,
   },
   'bsc-testnet': {
     chainId: 97,
     isTestnet: true,
-    explorerUrl: 'https://testnet.bscscan.com',
+    explorer: {
+      type: 'etherscan',
+      webUrl: 'https://testnet.bscscan.com',
+      apiUrl: 'https://api-testnet.bscscan.com/api',
+    },
     currency: Currency.tBNB,
   },
   'camp-network-testnet-v2': {
     chainId: 325000,
     isTestnet: true,
-    explorerUrl: 'https://camp-network-testnet.blockscout.com',
+    explorer: {
+      type: 'blockscout',
+      webUrl: 'https://camp-network-testnet.blockscout.com',
+      apiUrl: null,
```

Comment:
```suggestion
      apiUrl: 'https://camp-network-testnet.blockscout.com/api',
```

Reply:
This network is now over, I wonder if we should remove networks after sunset